deepblast/embedding.py

```diff
@@ -141,3 +141,47 @@ def forward(self, x):
             z = z.view(x.size(0), x.size(1), -1)
 
         return z
+
+
+class StackedCNN(nn.Module):
+    def __init__(self, nin, nembed, nout, k_size=5, nlayers=2, padding_idx=-1, dropout=0, sparse=False, lm=None):
+        super(StackedCNN, self).__init__()
+        self.conv = nn.Sequential(*([nn.Conv1d(in_channels=nembed if layer == 0 else nout,
+                                               out_channels=nout,
+                                               kernel_size=k_size,
+                                               padding=k_size//2) for layer in range(nlayers)]))
```

Comment:
```suggestion
                                               padding=k_size // 2) for layer in range(nlayers)]))
```
flake8 will complain otherwise.